src/Site.js

```diff
@@ -53,6 +54,25 @@ const LAYOUT_SCRIPTS_PATH = 'scripts.js';
 const LAYOUT_SITE_FOLDER_NAME = 'layouts';
 const USER_VARIABLES_PATH = '_markbind/variables.md';
 
+const SUPPORTED_BOOTSWATCH_THEMES = [
+  'cerulean',
+  'cosmo',
+  'flatly',
+  'journal',
+  'litera',
+  'lumen',
+  'lux',
+  'materia',
+  'minty',
+  'pulse',
+  'sandstone',
+  'simplex',
+  'sketchy',
+  'spacelab',
+  'united',
+  'yeti',
+];
```

Comment:
These correspond to subfolder names within bootswatch's `node_modules` folder.

Spelling of the themes are double checked with https://bootswatch.com/api/4.json